README.md

```diff
@@ -24,7 +24,7 @@ GENERAL|GENERAL_LOGOUT|Logout|Wyloguj|...
 Check out sample translation sheet [here](https://docs.google.com/spreadsheets/d/1HZwNTyo2XRkADjNLvXlMMlZ_buzCNYxhjIzvwL6mid8/)
 
 2. Click on **Share** button, and open **Advanced** panel
-3. Invite **<your account>@sheetstranslator.iam.gserviceaccount.com** with **view** permissions
+3. Invite **<your account>translator@sheetstranslator.iam.gserviceaccount.com** with **view** permissions
```

Comment:
I think it should be either:
`Invite **<your account>@sheetstranslator.iam.gserviceaccount.com** with **view** permissions`
or
`Invite **translator@sheetstranslator.iam.gserviceaccount.com** with **view** permissions`

not both 🤔

Reply:
Oh, I see my mistake